final_task/rss_reader/article.py

```diff
@@ -0,0 +1,87 @@
+from string_operations import *
+import logging
+import datetime
+from termcolor import colored
+
+
+
+class Article:
+    """single article class"""
+
+    def __init__(self, parsed, source):
+        """receive parsed article and extracts data from it"""
+        self.title = make_string_readable(parsed.title)
+        self.link = parsed.link
+        self.feed_link = source
+        self.published = extract_date(parsed)
+        summary_ = extract_topic_info_from_summary(parsed.summary)
+        self.summary = make_string_readable(summary_)
+        self.media = parsed.media_content
+
+        description_ = extract_image_info_from_summary(parsed.summary)
+        self.media_description = make_string_readable(description_)
+
+    def print_readable_article(self, is_colored):
```

Comment:
This function is large which lowers readability. It is better to split it somehow into the smaller ones.